doc/howtos/rdtraining/16_guidelines_pr.rst

```diff
@@ -0,0 +1,154 @@
+.. _howto/rdtraining/16_guidelines_pr:
+
+=========================================
+Part 16: Make It Shine And Submit Your PR
+=========================================
+
+In the previous chapters we saw how to create a fully functional business application. Now we will
+prepare our code, to be shared and learn how to share it.
+
+Coding guidelines
+=================
+
+We will start refactoring the code to match to the Odoo coding guidelines. The guidelines aim
+to improve the quality of Odoo Apps code.
+
+
+**Reference**: you will find the Odoo coding guidelines in :ref:`reference/guidelines`.
+
+.. exercise:: Make it shine
+
+    Refactor your code to respect the coding guidelines. Don't forget to respect the module
+    structure, the variable names, the method name convention, the model attribute order and the
+    xml ids.
+
+Your first Pull Request (PR)
+============================
+
+**The following is intended to be used by Odoo staff members only. All repositories mentioned are
+not accessible by third-party.**
+
+Now that your code respects the Odoo code guidelines and is shiny, let's share it with others.
+
+
+Configure development repository
+--------------------------------
+
+We will start creating a git development environment for the 'custom' folder, as we did it in the
+beginning for 'odoo' and 'enterprise'. The PR will target the `odoo/technical-training-sandbox` repository.
+
+To do this the first step is to
+`fork this repository in GitHub <https://guides.github.com/activities/forking/>`__ to create your
+own development repository.
```

Comment:
Doesn't really match the normal Odoo process though :smile: 

Also why two underscores on the link?